src/jmcomic/jm_plugin.py

```
@@ -799,6 +799,84 @@ def ensure_make_pdf_dir(pdf_dir: str):
         return pdf_dir
 
 
+class LongImgPlugin(JmOptionPlugin):
+    plugin_key = 'long_img'
+
+    def invoke(self,
+               photo: JmPhotoDetail = None,
```

Ruff

`JmPhotoDetail` may be undefined, or defined from star imports

(F405)